tket/include/tket/Circuit/Circuit.hpp

```diff
@@ -909,7 +909,7 @@ class Circuit {
   void add_bit(const Bit &id, bool reject_dups = true);
   register_t add_q_register(std::string reg_name, unsigned size);
   register_t add_c_register(std::string reg_name, unsigned size);
-  void add_wasm_register(std::size_t numer_of_w = 1);
+  void add_wasm_register(std::size_t number_of_w = 1);
```

Comment:
I think this was intended to be numer

Reply:
Why?

Should change in the implementation as well.